packages/core/src/transport.ts

```diff
@@ -35,6 +35,7 @@ export class Batch<T> {
   private upsertBuffer: { [key: string]: string } = {}
   private bufferBytesSize = 0
   private bufferMessageCount = 0
+  private biggerThanOneByteRegex = /([^\u0000-\u007F])/
```

Comment:
I double checked and corrected the initial `var regex = /([^\u0000-\u00ff])/` [https://en.wikipedia.org/wiki/UTF-8](https://en.wikipedia.org/wiki/UTF-8) 
<img width="650" alt="Capture d’écran 2020-06-03 à 17 29 59" src="https://user-images.githubusercontent.com/6943103/83656539-f66e9400-a5bf-11ea-95da-7b41b2a336e9.png">

Do you think. it would be suitable to link wikipedia ?

Reply:
why u00ff then?

Reply:
I corrected u00ff to u007F

Reply:
oh ok, link to wikipedia seems fine